### PR TITLE
app-backup/rsnapshot: version bump to 1.4.3

### PR DESCRIPTION
**Pull request by bor**
Also bump to EAPI version 7, drop old ebuilds
 and add myself back as maintainer

Closes: https://bugs.gentoo.org/711108
Signed-off-by: Sergiy Borodych <Sergiy.Borodych@gmail.com>

### CODE REVIEW

File: app-backup/rsnapshot/metadata.xml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!--maintainer-needed-->
+  <maintainer type="person">
+    <email>Sergiy.Borodych@gmail.com</email>
+    <name>Sergiy Borodych</name>
+  </maintainer>
```
Comment (hydrapolic) [inline app-backup/rsnapshot/metadata.xml:7]: Please re-add proxy-maint.

Comment (bor) [reply app-backup/rsnapshot/metadata.xml:7]: Yes, missed that.
Thank you for the point.
Added.

File: app-backup/rsnapshot/rsnapshot-1.4.3.ebuild [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-
-inherit eutils
+EAPI=7
 
 DESCRIPTION="A filesystem backup utility based on rsync"
 HOMEPAGE="http://www.rsnapshot.org"
 SRC_URI="http://www.rsnapshot.org/downloads/${P}.tar.gz"
```
Comment (juippis) [inline app-backup/rsnapshot/rsnapshot-1.4.3.ebuild:8]: `https://`

Comment (bor) [reply app-backup/rsnapshot/rsnapshot-1.4.3.ebuild]: Thanks, fixed

**Comment by bor** [comment]
Reworked into 2 commits.
Please check it.

**Pull request closed without merging**